docs/docs/install/docker-compose/index.md

```diff
@@ -75,28 +76,33 @@
 
 1. Set up the web server.
 
+    !!! info "Database Configuration" 
```

Comment:
I have a pedagogical/stylistic gripe with this new "Database Configuration" info box &mdash; a gripe which I've been having somewhat regularly with your docs content &mdash; so I want to take some extra time to explain my thinking.

I'll start with an analogy...

Let's pretend you're a tourist without a map in a city where I live, and you stop me on the sidewalk to ask for directions to the art museum.

- I could give you **reference information**, explaining how to navigate the city and where the art museum is located.

    > Within the downtown area, the streets run north/south, and the avenues run east/west. Streets have numbers which increase from west to east, and avenues are named alphabetically from north to south. Also the address numbers increase from west to east and from north to south. The art museum is at 5900 Elm Avenue.

- Or I could give you a **tutorial**, telling you exactly the steps you need to take to get to the art museum from your current location.

    > 1. Keep walking the way you're going.
    > 1. Take your next left on 4th Street and continue for a few blocks.
    > 1. Turn right on Elm Avenue and continue for one block.
    > 1. Look for the art museum on your left.

Each of those styles has helpful information in its own right. The tutorial style is more appropriate for someone who has a specific task at hand, like a tourist asking for directions; whereas the theoretical information would be more appropriate for someone who will need to perform similar tasks many different times, like a new delivery driver.

High quality software documentation has _both_ reference information, and tutorials. Our `configuration.md` page is a good example of appropriately written reference information. However, I think it's important to avoid intermingling the two documentation styles, because mixed styles can lead to content which is difficult to read and difficult to discover.

The installation docs should be written in tutorial form. They should be imperative as possible. Most sentences should begin with verbs.

When I read _"Set up the web server"_, I'm ready for an action. Good. But then I read _"By default, a Docker container named..."_ and I just hold my breath trying to figure out what I need to _do_. It's confusing and hard to read. Tutorials can hyperlink to reference information, and we might be able to utilize that strategy to improve this content. But I'm not sure exactly how to re-write this content because, even after reading it several times, I'm not sure what the relevant _action_ is for people installing Mathesar.

I had previously written the following step:

> 1. If you only plan to use Mathesar to connect to other preexisting Postgres database servers, then you may choose to [disable Mathesar's inbuilt Postgres database service](#disable-db-service) if you like. (Skip this step if you want to use Mathesar's Postgres database service.)

I see that you've moved that content into the end of the info box. I think it would be better to keep that content as a step, so that the _action_ is clearer to the reader. Then, if there are more details to explain, we could either do that in the reference documentation (under the "Start Mathesar without the database server" heading), or we could try to explain it more imperatively within the tutorial.

Do you think you could take a stab at addressing this concern? I'm happy to hop on a chat or call with you to talk it through more if that would help.